src/njsVariable.c

```diff
@@ -531,6 +542,10 @@ bool njsVariable_initForQuery(njsVariable *vars, uint32_t numVars,
             case DPI_ORACLE_TYPE_NATIVE_DOUBLE:
             case DPI_ORACLE_TYPE_ROWID:
                 break;
+            case DPI_ORACLE_TYPE_BOOLEAN:
+                vars[i].varTypeNum = DPI_ORACLE_TYPE_BOOLEAN;
+                vars[i].nativeTypeNum = DPI_NATIVE_TYPE_BOOLEAN;
+                break;
```

Comment:
This section is not needed as booleans cannot be fetched. They can only be used in PL/SQL.